server/helpers/email.helper.js

```diff
@@ -0,0 +1,52 @@
+const AWS = require('aws-sdk');
+
+const accessKey = process.env.AWS_SES_ACCESS_KEY;
+const secretKey = process.env.AWS_SES_SECRET_KEY;
+const awsRegion = process.env.AWS_SES_REGION;
```

Comment:
Need to create `server/config/aws.config.js`

Reply:
fixed